subsys/emds/emds.c

```diff
@@ -6,55 +6,353 @@
 
 #include <emds/emds.h>
 
+#include <zephyr.h>
+#include <storage/flash_map.h>
+#include <drivers/flash.h>
+#include <emds/emds_flash.h>
```

Comment:
```suggestion
#include "emds_flash.h"
```
Changes after merge of https://github.com/Balaklaka/sdk-nrf/pull/5